src/runtime/String.h

```diff
@@ -88,6 +88,9 @@ class StringWriteOption {
 };
 
 struct StringBufferAccessData {
+    // should be allocated on the stack
```

Comment:
Q. Why did you make this decision?

Reply:
`StringBufferAccessData` is designed to be allocated on the stack.
This structure is created for each string access, so temporally used and then reclaimed.
Therefore, `StringBufferAccessData` is defined as just a normal structure not GC object.
I added this comment to clarify its usage.